actions.yaml

```diff
@@ -24,3 +25,7 @@ upgrade-packages:
             description: |-
                 Upgrade gpu drivers in the event there are package updates available.
                 Note: gpu drivers are ignored if gpu_support is unecessary
+        force:
+            type: boolean
+            default: false
+            description: Only works if `gpu=true`. Force reinstall the gpu drivers.
```

Comment:
why is force only for `gpu=True`?  Why doesn't it support all the packages?

Reply:
We can, though I've never heard of a scenario where someone wanted to force containerd to reinstall.. With the situation I've gotten into though I can see a scenario where we would want to force the nvidia drivers. If you'd like I can change it though.